beetsplug/acousticbrainz.py

```diff
@@ -41,64 +45,161 @@ def func(lib, opts, args):
         cmd.func = func
         return [cmd]
 
+    def import_task_files(self, session, task):
+        """Function is called upon beet import.
+        """
+
+        items = task.imported_items()
+        fetch_info(self._log, items)
+
 
 def fetch_info(log, items):
-    """Currently outputs MBID and corresponding request status code
+    """Currently outputs MBID and corresponding request status code.
     """
     for item in items:
         if item.mb_trackid:
```

Comment:
Rather than indenting the whole rest of the loop, you could invert this and use `continue`.